static/scss/flex/article.scss

```diff
@@ -1,17 +1,6 @@
 article {
-    display: -webkit-box;
-    display: -ms-flexbox;
     display: flex;
-    -webkit-box-flex: 1;
-    -ms-flex: 1;
-    flex: 1;
-}
-
-
-
-article section.page {
```

Comment:
`section.page` did not appear anywhere in the generated html.

Moved all the relevant css to apply to the `article`, which is the main content area on the page